packages/devtools_app/lib/src/screens/performance/panes/timeline_events/perfetto/_perfetto_controller_web.dart

```diff
@@ -99,7 +99,7 @@ class PerfettoControllerImpl extends PerfettoController {
   ///
   /// See [_viewIdIncrementer] for an explanation of why we use an incrementer
   /// in the id.
-  late final viewId = 'embedded-perfetto-${_viewIdIncrementer++}';
```

Comment:
remove type on left hand side since it is inferred

Reply:
This seems inconsistent across DevTools. Is this preference documented somewhere?

Many other Dart team projects will leave the type for fields since they end up being part of the class's API.

Reply:
This isn't documented anywhere that I am aware of, but this is a convention we have followed in the DevTools codebase for a long time. Mirroring more Dart projects as opposed to Flutter, which requires the types on the left and the right. The argument is that it makes the code unnecessarily verbose. 

We do have some style conventions documented [here](https://github.com/flutter/devtools/blob/master/STYLE.md), but preferring types on the RHS only where possible is not documented there (it probably should be).